entries/z/zip.co.json

```diff
@@ -0,0 +1,21 @@
+{
+	"Zip": {
+		"domain": "zip.co",
+		"tfa": [
+			"sms"
+		],
+		"notes": "2FA is only used when an account login is made from an unrecognised device; user cannot enable 2FA for every login.",
```

Comment:
Usually a device is identified by a cookie, so deleting the cookies may reset the state to "unrecognised". In that case, we don't really need this note, I reckon.
```suggestion
```